packages/toolkit/src/mapBuilders.ts

```diff
@@ -159,6 +159,11 @@ export function executeReducerBuilderCallback<S>(
         typeof typeOrActionCreator === 'string'
           ? typeOrActionCreator
           : typeOrActionCreator.type
+      if (!type) {
+        throw new Error(
+          '`builder.addCase` cannot be called with an empty action type'
```

Comment:
I was torn on whether to use just `addCase` to match the actionsMap error or to use `builder.<method>` to match the pattern of the rest of the error messages in this file

Reply:
matching the rest makes sense - in fact it would be good to update the actionsMap error to match too

Reply:
Happy to do that! Would you like me to create a separate pr?

Reply:
nah, it seems like it would be fine to do here imo

Reply:
looks good to me - slightly odd that the CI tests haven't run though

Reply:
Weird! Anything I can do to help?